dev-libs/nspr/nspr-4.26.recipe

```diff
@@ -4,21 +4,26 @@ Netscape Portable Runtime (NSPR) provides a platform-neutral API for system \
 level and libc-like functions. The API is used in the Mozilla clients, many \
 of Red Hat's and Sun's server applications, and other software offerings.
 "
+HOMEPAGE="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR"
+COPYRIGHT="Mozilla Foundation"
 LICENSE="
 GNU GPL v2
 GNU LGPL v2.1
 MPL v2.0
 "
-COPYRIGHT="Mozilla Foundation"
-HOMEPAGE="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR"
-SOURCE_URI="https://ftp.mozilla.org/pub/mozilla.org/nspr/releases/v$portVersion/src/nspr-$portVersion.tar.gz"
-CHECKSUM_SHA256="343614971c30520d0fa55f4af0a72578e2d8674bb71caf7187490c3379523107"
 REVISION="1"
-ARCHITECTURES="!x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86"
+SOURCE_URI="https://ftp.mozilla.org/pub/mozilla.org/nspr/releases/v$portVersion/src/nspr-$portVersion.tar.gz"
+CHECKSUM_SHA256="fc9d142d85b74ffd2e6374a0c9016f3f2dac074225e24df3070e5a72d31b773d"
 SOURCE_DIR="nspr-$portVersion/nspr"
+if [ $effectiveTargetArchitecture == 'x86_64' ]; then
+		PATCHES="nspr-$portVersion.patchset"
+else
+		PATCHES="nspr_x86-$portVersion.patchset"
+fi
```

Comment:
At first look I don't see any difference in both patchsets, if so, only 1 should be needed

Reply:
No need to use seperate patchsets here ...

Reply:
Fixed